apps/opik-frontend/src/components/pages/PromptPage/PromptTab/PromptTab.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useRef, useState } from "react";
+
+import { Button } from "@/components/ui/button";
+import { Info, Pencil } from "lucide-react";
+import { PromptWithLatestVersion } from "@/types/prompts";
+import Loader from "@/components/shared/Loader/Loader";
+import usePromptVersionsById from "@/api/prompts/usePromptVersionsById";
+import UseThisPromptDialog from "@/components/pages/PromptPage/PromptTab/UseThisPromptDialog";
+import EditPromptDialog from "@/components/pages/PromptPage/PromptTab/EditPromptDialog";
+import CommitHistory from "@/components/pages/PromptPage/PromptTab/CommitHistory";
+import usePromptVersionById from "@/api/prompts/usePromptVersionById";
+import { StringParam, useQueryParam } from "use-query-params";
+
+interface PromptTabInterface {
+  prompt?: PromptWithLatestVersion;
+}
+
+const PromptTab = ({ prompt }: PromptTabInterface) => {
+  const [openUseThisPrompt, setOpenUseThisPrompt] = useState(false);
+  const [openEditPrompt, setOpenEditPrompt] = useState(false);
+
+  const [activeVersionId, setActiveVersionId] = useQueryParam(
+    "activeVersionId",
+    StringParam,
+    {
+      updateType: "replaceIn",
+    },
+  );
+
+  const editPromptResetKeyRef = useRef(0);
+
+  const { data } = usePromptVersionsById(
+    {
+      promptId: prompt?.id || "",
+      page: 1,
```

Comment:
So we won't have any sort of pagination in this component to load more in this page, right?

Reply:
@ferc we have the same data in `Commits tab` 
here we will show first 25 commits